### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by d4nQw3rty**
In this branch aI will Develop just the UI of the calculator.
### General requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make a correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that the work is documented [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- Follow Microverse's list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)
### Project requirements

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file
- In Calculator.js, create a React component that matches this design:
image
- Use class-based components
- Once the Calculator is ready, import it and use it in your main component (App.js).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (BenMukebo) [inline src/App.js]: - [ ] Great job so far :clap However According to the project requirements The App.js component should also be refactored into class-based. 




```ruby
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @d4nQw3rty,


You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️
- The Calculator component has been well refactored to the class-based component  👍🏻 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (d4nQw3rty) [reply src/App.js]: Oh I am sorry @BenMukebo Sorry, I understood all components in the folder components, I did not think that I should change the main App, I will work on it :muscle:

**Review by DeliceLydia — APPROVED** [review]
## Hi @d4nQw3rty, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**